src/encoded/tests/test_submit.py

```diff
@@ -1292,6 +1294,71 @@ def test_xls_to_json_pedigree_errors(testapp, project, institution):
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-1', '', 'Fam1', 'Person1-1', 'M', '20', 'year', '', 'proband', 'peripheral blood', 'Specimen1-1', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-2', 'Case1', 'Fam1', 'Person1-2', 'F', '50', 'year', '', 'mother', 'peripheral blood', 'Specimen1-2', 'N', 'WGS', '', '', '']], "Row 3", "Row 5"),
+        ([['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Unique Analysis ID*:', 'Analysis ID*:', 'Family ID:', 'Individual ID*:', 'Sex*:', 'Age:', 'Age Units:', 'Birth Year:', 'Relation to Proband*:', 'Specimen Type*:', 'Specimen ID*:', 'Report Required*:', 'Test Requested*:', 'Specimen Collection Date:', 'Sequencing Date:', 'Files:'],
+            ['Case1_Person1-1', '', 'Fam1', 'Person1-1', 'M', '20', 'year', '', 'proband', 'peripheral blood', 'Specimen1-1', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-2', 'Case1', 'Fam1', 'Person1-2', 'F', '50', 'year', '', 'mother', 'peripheral blood', 'Specimen1-2', 'N', 'WGS', '', '', '']], "Row 3", "Row 5"),
+        ([['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Unique Analysis ID*:', 'Analysis ID*:', 'Family ID:', 'Individual ID*:', 'Sex*:', 'Age:', 'Age Units:', 'Birth Year:', 'Relation to Proband*:', 'Specimen Type*:', 'Specimen ID*:', 'Report Required*:', 'Test Requested*:', 'Specimen Collection Date:', 'Sequencing Date:', 'Files:'],
+            ['Case1_Person1-1', '', 'Fam1', 'Person1-1', 'M', '20', 'year', '', 'proband', 'peripheral blood', 'Specimen1-1', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-2', 'Case1', 'Fam1', 'Person1-2', 'F', '50', 'year', '', 'mother', 'peripheral blood', 'Specimen1-2', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']], "Row 3", "Row 5")
+    ])
+def test_xls_to_json_accession_row_counting(testapp, project, institution, digested_xlsx, expected_error_row_number_1, expected_error_row_number_2):
+    rows = (row for row in digested_xlsx) # make generator out of the digested_xlsx parameter
+    json_out, success = xls_to_json(testapp, rows, project, institution, TEST_INGESTION_ID1, 'accessioning')
+    joined_errors = "".join(json_out["errors"])
+    assert expected_error_row_number_1 in joined_errors
+    assert expected_error_row_number_2 in joined_errors
+    assert success
```

Comment:
Consider re-factoring to not having 2 expected row numbers, as well as including a parametrize call that has no errors. Reasons for this are twofold:

- You should catch different types of errors in separate parametrize calls (and probably document which error exists for which call with a quick comment to make it more obvious). You can then generate another call to mix different errors together if you feel it's necessary.
- Requiring 2 errors for parametrize call is arbitrary and forces the next person refactoring to either refactor the test or include 2 errors when they may want to have 1 (or 3)

Also, the comment is not adding much here.
```suggestion
def test_xls_to_json_accession_row_counting(testapp, project, institution, digested_xlsx, expected_error_rows):
    rows = (row for row in digested_xlsx)
    json_out, success = xls_to_json(testapp, rows, project, institution, TEST_INGESTION_ID1, 'accessioning')
    joined_errors = "".join(json_out["errors"])
    for row_with_error in expected_error_rows:
         assert f"Row {row_with_error}" in joined_errors
    assert success
```